tests/test_converter.py

```diff
@@ -0,0 +1,102 @@
+"""Unit test for converter."""
+
+from pathlib import Path
+import numpy as np
+import pytest
+import xarray as xr
+from zampy.datasets import EthCanopyHeight
+from zampy.datasets import converter
+from zampy.datasets.eth_canopy_height import parse_tiff_file
+
+
+path_dummy_data = Path(__file__).resolve().parent / "test_data" / "eth-canopy-height"
```

Comment:
You can use 
```
from test_datasets import data_folder
path_dummy_data = data_folder / "eth-canopy-height"
```

Reply:
I couldnot understand why there is a folder `test_datasets` but the tests for datasets are located outside of this folder :thinking:

Reply:
Thanks for the suggestion! This looks much cleaner now. 

> I couldnot understand why there is a folder `test_datasets` but the tests for datasets are located outside of this folder 🤔

I simply want to organize the test folder a bit. We will add a few other datasets (incl. era5, era5 land, lai, land-cover, dem, etc.), which we can put them in `test_datasets` folder. And for other dataset related utils files/functions, they can be placed outside. I think this makes it more clear. But let's discuss it then.